### DISCUSSION TITLE
In Millions Display

### DISCUSSION THREAD

**Discussion by envitraux** [dm4t2/vue-currency-input#276] [Ideas]
I was wondering if there was way to have the input automatically scale the numbers down to in millions for display and entry, but when the model updates the value will be in millions.

Example: a modelValue of $12,300,000 would be displayed as $12.3

If a user enters: $345.8 the modelValue  would be $345,800,000

I know this is a simple scaling, but I wanted to see if I could have the component have a scale prop where 6 would scale it from millions or something like that.

**Comment by envitraux** [comment]
I know how I am going to do it, but still would love some insight. I will make a component that wraps around the vue-currency-input component, that will handle that. That will also enable for me to allow it to flip between 2 currencies which I need for my app.

**Comment by dm4t2** [comment]
Hi, thanks for your suggestion 👍 

There's already a precision dependent value scaling when using the `exportValueAsInteger` option, for example: `0.01` -> `100`

I think this behavior can be combined with a new `valueScaling` option which accepts the enum values

- `precision` 
- `thousands`
- `millions`
- `billions` 

This will make the  `exportValueAsInteger` option deprecated.

**Reply by envitraux** [reply]
Has this been implemented?

**Reply by dm4t2** [reply]
It's still in progress and will be shortly released with v2.3.0.

**Reply by dm4t2** [reply]
I have created an issue: #305
